src/popup/index.js

```diff
@@ -50,3 +51,14 @@ browser.tabs.query({ currentWindow: true, active: true })
     store.domain = tld.getDomain(domain) || domain;
   }
 });
+
+function waitForBody() {
```

Comment:
This function can be removed if Vue is created and mounted inside browser.tabs.query() callback.